### PR TITLE
Feature/session middleware refactoring

### PR DESCRIPTION
**Pull request by eugenyProchan**
1. Have simplified `SessionMiddlware` class, change his responsibility only session. 
2. Have renamed `sessionMiddlware` to `SessionMiddlware`.
3. Have deleted `auth` endpoint.
4. Have Added TODO comment in `SessionMiddlware`, perhaps we need change `request.state.player` to `request.state.session`. But until I don't become to do it.

### CODE REVIEW

File: src/middleware.py [hunk 1]
```diff
@@ -1,51 +1,22 @@
-from fastapi import Request, status
-from fastapi.responses import Response, JSONResponse
+import uuid
+from fastapi import Request
```
Comment (D-Mantrid) [inline src/middleware.py:3]: A good practice is to divide three lavels of import with blank lines in the following order:
- standard library modules
- third-party packages
- internal project modules

File: src/main.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-from src.middleware import sessionMiddleware
+from src.middleware import SessionMiddleware
 from src.routes import (
     moving_router,
     actions_router,
-    auth_router,
     prepare_game_router
 )
+
 from fastapi import FastAPI
```
Comment (D-Mantrid) [inline src/main.py]: See the comment above about importing codestyle.

File: src/middleware.py [hunk 3]
```diff
@@ -1,51 +1,22 @@
...
 from starlette.middleware.base import BaseHTTPMiddleware
-import jwt
 
 
-SECRET_KEY = 'you will never guess'
-
-
-class sessionMiddleware(BaseHTTPMiddleware):
+class SessionMiddleware(BaseHTTPMiddleware):
     def __init__(self, app):
```
Comment (D-Mantrid) [inline src/middleware.py]: If we don't have plans to expand `__init__` functionality of this class, we can omit this declaration.

File: src/middleware.py [hunk 4]
```diff
@@ -1,51 +1,22 @@
...
+        response = await call_next(request)
 
         if not session:
-            response = JSONResponse({'error': 'Unauthorized'}, status_code=status.HTTP_401_UNAUTHORIZED)
-            response.headers['WWW-Authentication'] = 'URI /auth'
-            return response
-        else:
-            request.state.player = _get_player_uuid(session)
-            response = await call_next(request)
+            response.set_cookie(key='session', value=str(uuid.uuid4()))
```
Comment (D-Mantrid) [inline src/middleware.py:19]: I think we can start upgrading session structure from now:
```suggestion
            session = {
                'id': str(uuid.uuid4()),
            }
            response.set_cookie(key='session', value=session)
```

**Review by D-Mantrid — CHANGES_REQUESTED** [review]
I left small requests for codestyle improvements and one suggestion.

File: src/middleware.py (hunk 1 above)
Comment (eugenyProchan) [reply src/middleware.py:3]: I did't know about it, only about order.

File: src/middleware.py (hunk 4 above)
Comment (eugenyProchan) [reply src/middleware.py:19]: Do you think, we need `id` key for any purposes?

Comment (D-Mantrid) [reply src/middleware.py:19]: Well, we could use it as "player_id", but the main reason of my suggestion is that we need a session which is able to hold bigger data, not only player ID (game ID, metadata, or whatever). For example, session objects in Flask applications are always mapping objects.

Comment (D-Mantrid) [reply src/middleware.py:19]: We decided to resolve this question later

**Review by D-Mantrid — APPROVED** [review]

**Pull request merged**